command/pr_create.go

```diff
@@ -68,6 +101,11 @@ func prCreate(cmd *cobra.Command, _ []string) error {
 		return fmt.Errorf("could not parse body: %w", err)
 	}
 
+	defs, err := computeDefaults(baseBranch, headBranch)
+	if err != nil {
+		return fmt.Errorf("could not compute title or body defaults:  %w", err)
```

Comment:
I wonder if we should warn but not abort the operation in this case? If fetching the git commits failed for some reason, we can still give a person the chance to fill in title & body by themselves.

Reply:
makes sense.